### PR TITLE
docs(migration): add Card component migration guide

### PR DESCRIPTION
**Pull request by massao**

### CODE REVIEW

**Review by gui-santos — APPROVED** [review]

File: packages/forma-36-website/src/content/migration-v3-to-v4.mdx [hunk 1]
```diff
@@ -28,6 +28,8 @@ To migrate your project easily, we provided you with a codemod for most of the c
 
 - **Button** API improvements: Some properties of the Button components were migrated or changed. For more information, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-button-components).
 
+- **Card** Some properties of the Card components were migrated or changed. For more information, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-card-components).
```
Comment (burakukula) [inline packages/forma-36-website/src/content/migration-v3-to-v4.mdx:31]: Maybe it's worth mentioning that the guide contains how to migrate `Card`, `EntryCard` and `AssetCard` ?

**Review by burakukula — APPROVED** [review]

File: MIGRATION.md [hunk 2]
```diff
@@ -259,9 +261,178 @@ import {
 
 You can also make these changes manually.
 
+### Card
+
+In v4, the Card components has received API improvements. The changes are based on our code style guide, which create consistent, easy to use APIs. For example:
```
Comment (alstaicu) [inline MIGRATION.md:266]:
```suggestion
In v4, the Card component has received API improvements. The changes are based on our code style guide, which create consistent, easy to use APIs. For example:
```

File: MIGRATION.md [hunk 3]
```diff
@@ -259,9 +261,178 @@ import {
...
+  actions={[<Menu.Item>action</Menu.Item>]}
+  size="default" />;
+```
+
+This is an overview of the changed props:
+
+- `cardDragHandleCopmonent` was removed
+- `selected` was renamed to `isSelected`
+- `isDragActive` was renamed to `isDragging`
+- `statusIcon` was renamed to `icon`, and now expect an Icon component
```
Comment (alstaicu) [inline MIGRATION.md:347]:
```suggestion
- `statusIcon` was renamed to `icon`, and now expects an Icon component
```

File: MIGRATION.md [hunk 4]
```diff
@@ -259,9 +261,178 @@ import {
...
+- `cardDragHandleProps` was renamed to `dragHandleProps`
+- `dropdownListElements` was updated to use the new `Menu` component and should receive an array of `Menu.Items`, check the Menu [documentation](https://v4-forma-36.netlify.app/components/menu/) for more info.
```
Comment (alstaicu) [inline MIGRATION.md:349]:
```suggestion
- `dropdownListElements` was updated to use the new `Menu` component and should receive an array of `Menu.Items`. For more information, see the Menu [documentation](https://v4-forma-36.netlify.app/components/menu/).
```

File: MIGRATION.md [hunk 5]
```diff
@@ -259,9 +261,178 @@ import {
...
+
+#### How to migrate your Card component
+
+To migrate the `Card` component to the v4 run the following [codemod](https://github.com/contentful/forma-36/tree/forma-v4/packages/forma-36-codemod):
```
Comment (alstaicu) [inline MIGRATION.md:353]:
```suggestion
To migrate the `Card` component to v4, run the following [codemod](https://github.com/contentful/forma-36/tree/forma-v4/packages/forma-36-codemod):
```

File: MIGRATION.md [hunk 6]
```diff
@@ -259,9 +261,178 @@ import {
...
+
+`npx @contentful/f36-codemod`
+
+We have codemods for `Card`, `AssetCard` and `EntryCard`, when running the codemods, the following changes occur:
```
Comment (alstaicu) [inline MIGRATION.md:357]:
```suggestion
There are codemods for `Card`, `AssetCard` and `EntryCard`. When running the codemods, the following changes occur:
```

File: packages/forma-36-website/src/content/migration-v3-to-v4.mdx [hunk 7]
```diff
@@ -28,6 +28,8 @@ To migrate your project easily, we provided you with a codemod for most of the c
 
 - **Button** API improvements: Some properties of the Button components were migrated or changed. For more information, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-button-components).
 
+- **Card, AssetCard, and EntryCard** We changed some properties of the Card, AssetCard, and EntryCard components were migrated or changed. For more information, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-card-components).
```
Comment (alstaicu) [inline packages/forma-36-website/src/content/migration-v3-to-v4.mdx:31]:
```suggestion
- **Card, AssetCard, and EntryCard** : Some properties of the Card, AssetCard, and EntryCard components were migrated or changed. For more information, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-card-components).
```

**Pull request merged**